image/image.go

```diff
@@ -244,9 +244,6 @@ func installCloudConfig(gadgetDir string) error {
 	if osutil.FileExists(cloudConfig) {
 		dst := filepath.Join(cloudDir, "cloud.cfg")
 		err = osutil.CopyFile(cloudConfig, dst, osutil.CopyFlagOverwrite)
-	} else {
```

Comment:
Do we want to remove this file from existing installations?

Reply:
I suspect no.

Existing users have either manually removed the file (or passed in cloud-config when building an image) so I don't expect there are existing images which would now want to run cloud-init.

If we did remove the file, users who weren't utilizing cloud-init won't see a change (cloud-init won't enable itself unless it's needed).

However, users who have a nocloud-net seed in /var/lib/cloud/seed, from early snappy times or are running the image on a cloud (but were not using cloud-init) would now have cloud-init run where it wasn't before.

I can't think of a scenario where it wouldn't be surprising if we remove it from existing images.

We may still have some users who want to forcibly disable cloud-init (testing an image on a cloud for dev/test but targeting a IOT device);   We likely want to point users at the cloud-init docs[1] which discuss how cloud-init can be disabled and possibly a forum topic/article.

1. http://cloudinit.readthedocs.io/en/latest/topics/boot.html#generator